src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -647,21 +647,9 @@ static bool _hslToRgb(float hue, float satuation, float brightness, uint8_t* red
         }
     }
 
-    i = static_cast<uint8_t>(_red * 255.0);
-    f = (_red * 255.0) - i;
-    _red = (f <= 0.5) ? i : (i + 1);
-
-    i = static_cast<uint8_t>(_green * 255.0);
-    f = (_green * 255.0) - i;
-    _green = (f <= 0.5) ? i : (i + 1);
-
-    i = static_cast<uint8_t>(_blue * 255.0);
-    f = (_blue * 255.0) - i;
-    _blue = (f <= 0.5) ? i : (i + 1);
-
-    *red = static_cast<uint8_t>(_red);
-    *green = static_cast<uint8_t>(_green);
-    *blue = static_cast<uint8_t>(_blue);
+    *red = static_cast<uint8_t>(ceil(_red * 255.0f));
```

Comment:
why ceil()? roundf() seems to be correct.

Reply:
by mistake. thanks.